content/departments/product-engineering/engineering/cloud/delivery/managed/upgrade_process.md

```diff
@@ -445,3 +452,85 @@ For reference, the above steps were adapted from [Working with persistent disks]
 3. [Take down the old deployment](#10-take-down-the-old-deployment)
 4. [Remove the maintainence banner](#11-remove-the-banner-indicating-maintenance-is-in-progress)
 5. [Open a pull request](#13-open-a-pull-request-to-commit-your-changes)
+
+## In-place updates
+
+This is the upgrade process for running in-place updates. This approach is riskier than the above processes, because rolling back changes takes significantly longer.
+
+Only use this approach for low-risk patch upgrades or docker-compose container resource changes. **Do not use if the patch includes a database change.**
+
+### 0) Upgrade setup
+
+**Follow the [general setup](#general-setup) guide.** Then redefine the NEW_DEPLOYMENT variable to match the existing deployment:
+
+```sh
+export NEW_DEPLOYMENT=$OLD_DEPLOYMENT
+```
+
+Validate all variables are set:
+
+```sh
+./util/validate-env.ts
+```
+
+Then set up a branch for your changes:
+
+```sh
+git checkout -b $CUSTOMER/upgrade to $NEW_VERSION in-place
+# all the below steps are documented assuming you are in the customer deployment directory
+cd $CUSTOMER
+```
+
+### 1) Create a snapshot
+
+Create a snapshot:
+
+```sh
+../util/create-snapshot.ts $OLD_DEPLOYMENT upgrade-machine-$DATE
```

Comment:
```suggestion
../util/create-snapshot.ts $OLD_DEPLOYMENT
```

the default generated name for snapshots will work here, same as https://github.com/sourcegraph/handbook/blob/document-inplace-upgrades/content/departments/product-engineering/engineering/cloud/delivery/managed/upgrade_process.md#3-create-a-snapshot-of-the-current-deployment

Reply:
TY! Bad copy-paste + editing on my part.